src/main/java/de/tum/in/www1/artemis/service/scheduled/ProgrammingExerciseScheduleService.java

```diff
@@ -274,7 +278,6 @@ private void scheduleTemplateCommitCombination(ProgrammingExercise exercise) {
     }
 
     private void scheduleDueDateLockAndScoreUpdate(ProgrammingExercise exercise) {
-        // no rebuild date is set but test cases marked with AFTER_DUE_DATE exist: they have to become visible by recalculation of the scores
```

Comment:
Apparently, this comment is not helpful. Can you maybe replace it with a better one?

Reply:
I just move it the the method itself (and not above the method call) since I think it makes more sense there.